exercises/practice/anagram/AnagramTest.php

```diff
@@ -15,7 +15,7 @@ public static function setUpBeforeClass(): void
      */
     public function testNoMatches(): void
     {
-        $this->assertEquals([], detectAnagrams('diaper', ['hello', 'world', 'zombies', 'pants']));
+        $this->assertEqualsCanonicalizing([], detectAnagrams('diaper', ['hello', 'world', 'zombies', 'pants']));
```

Comment:
To really make `assertEqualsCanonicalizing()` work, `array_values()` also is required in case of non-linear indexes, as that would cause PHPUnit to think indexes matter:

```suggestion
        $this->assertEqualsCanonicalizing([], array_values(detectAnagrams('diaper', ['hello', 'world', 'zombies', 'pants'])));
```